system/Common.php

```diff
@@ -1128,7 +1128,7 @@ function timer(?string $name = null, ?callable $callable = null)
     {
         $timer = service('timer');
 
-        if ($name === null) {
+        if ((string) $name === '') {
```

Comment:
This is a change of behavior. You can have a timer named `''`.

Reply:
We can still get the value through an empty key `$timers[""]`, but it looks weird.

Reply:
Good. I will cancel the update